src/App.js

```diff
@@ -1,16 +1,133 @@
 import React, { Component } from 'react'
-import logo from './logo.svg'
 import './App.css'
+import PlayMenu from './PlayMenu'
+import Naming from './Naming'
 
 class App extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      game: {
+        gameState: 'naming',
+        difficulty: 1
+      },
+      progress: {
+        days: 12,
+        miles: 123
+      },
+      inventory: {
+        waterFilter: 12,
+        sleepingBags: 3,
+        food: 200
+      },
+      people: [{
+        name: 'You',
+        health: 100,
+        status: 'alive'
+      }, {
+        name: '',
+        health: 100,
+        status: 'dead'
+      }, {
+        name: '',
+        health: 100,
+        status: 'alive'
+      },
+      {
+        name: '',
+        health: 100,
+        status: 'alive'
+      },
+      {
+        name: '',
+        health: 100,
+        status: 'alive'
+      }
+      ]}
+    this.onUserPlay = this.onUserPlay.bind(this)
+    this.handleName = this.handleName.bind(this)
+    this.onConfirm = this.onConfirm.bind(this)
+  }
+
+  rangeGenerator (lowest, highest) {
+    const min = Math.ceil(lowest)
+    const max = Math.floor(highest)
+    return Math.floor(Math.random() * (max - min)) + min
+  }
```

Comment:
`rangeGenerator` has nothing to do with this component and does not use `this`. I would therefore tend to take a method like this and pull it all the way out of the class, either above it or into a `utility.js` file.